src/main/webapp/dev/js/main/instructorFeedbackEdit.es6

```diff
@@ -1087,7 +1087,11 @@ function readyFeedbackEditPage() {
     setupQuestionCopyModal();
 
     // Additional formatting & bindings.
-    disableEditFS();
+    if ($('#form_feedbacksession').data(`${ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT}`) === true) {
```

Comment:
@whipermr5 Yes, and it would then load the page with edit mode disabled. That being said, there is no way for this to be empty in the current scenario. This `instructorFeedbackEdit.es6` is loaded only for the `instructorFeedbackEditPage`.

The property will be empty only on the `instructorFeedbackSessionsPage`, whose JS file has no dependence on the attribute.